### DISCUSSION TITLE
1.5.5

### DISCUSSION THREAD

**Discussion by guqing** [halo-dev/halo#2543] [General]
## Improvements
- 完善install路径返回的页面结果。#2225 @cetr 
- 修改根据分类查询文章方法，使其同时返回子分类文章。#2405 @JustinLiang522

## Bug Fixes
- 修复七牛云做附件存储时附件无法上传非图片文件的问题。#2331 @AirboZH
- 修复了当云存储中文件被删除时，在 Halo 同时删除多个附件会失败的问题。#2317 @JustinLiang522
- 修复文章详情页 meta_description 内容生成为空的问题。#2282 @guqing 
- 修复了无法设置日志、相册、链接页标题和每页显示条数的问题。halo-dev/console#601 @JustinLiang522

**Full Changelog**: https://github.com/halo-dev/halo/compare/v1.5.4...v1.5.5

<hr /><em>This discussion was created from the release <a href='https://github.com/halo-dev/halo/releases/tag/v1.5.5'>1.5.5</a>.</em>

**Comment by ql0202cocou** [comment]
有1.6.0，还有1.5.5啊？

**Reply by JohnNiang** [reply]
1.5.x 还得维护一段时间。不过只会修复 bug，没有功能。

**Reply by ql0202cocou** [reply]
好的，谢谢各位开发大大。
